tests/qui_widgets_update.pm

```diff
@@ -41,7 +41,6 @@ sub run {
 
     # launch the updater
     assert_and_click('qubes-update-launch-updater', timeout => 20);
-    assert_and_click('qubes-update-enable-for-all', timeout => 20);
```

Comment:
This step is still necessary in 4.1, so it should be guarded with `if (check_var("VERSION", "4.1"))` instead.

Reply:
done